app/src/main/java/com/azuresamples/msalnativeauthandroidkotlinsampleapp/MFAFragment.kt

```diff
@@ -116,6 +116,8 @@ class MFAFragment : Fragment() {
                     handleSignInError(actionResult)
                 }
                 else -> {
+                    // Please refer to other sign in fragments for the handling of other possible results.
```

Comment:
I'd just say:
```suggestion
                    // Unexpected result
```
As you did in other else sections

Reply:
The comment here guide developer the route and instructions when they meet unexpected_sdk_result